Dockerfile

```diff
@@ -16,4 +16,7 @@ RUN \
   apt-get remove -y git && \
   apt-get -y autoremove
 
-ENTRYPOINT bin/iotagent-json config.js
+RUN npm install pm2@latest -g
```

Comment:
Could this npm install command be included in the first RUN (to save container size, as each RUN involves an extra layer in the image)?